docker/Dockerfile

```diff
@@ -14,6 +14,9 @@ ENV PDB2PQR='/app/pdb2pqr/pdb2pqr'
 
 ARG CACHEBUST
 COPY requirements.txt .
+COPY tests .
+COPY run_tests.py .
+
```

Comment:
```suggestion
```
These lines aren't required due to the `COPY . .` instruction below. Just the change to `.dockerignore` is necessary